webviz_subsurface/plugins/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -243,10 +243,10 @@ def ensemble_selector(self) -> html.Div:
                         id=self.ids("ensemble"),
                         options=[
                             {"label": i, "value": i}
-                            for i in list(self.data["ENSEMBLE"].unique())
+                            for i in list(self.parameters["ENSEMBLE"].unique())
                         ],
                         clearable=False,
-                        value=list(self.data["ENSEMBLE"].unique())[0],
+                        value=list(self.parameters["ENSEMBLE"].unique())[0],
```

Comment:
Not duplication, but hunting for everything now 😉
Could maybe move the `list(self.parameters["ENSEMBLE"].unique())` to somewhere else like `ensembles=list(self.parameters["ENSEMBLE"].unique())` before the return of this function, instead of calling it once for the options, and once for the initial value?